code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -113,7 +113,7 @@
 	volume = 100
 	list_reagents = list("nanites" = 100)
 
-/obj/item/reagent_containers/hypospray/autoinjector
+/obj/item/reagent_containers/hypospray/autoinjector // This is an empty variant
 	name = "emergency autoinjector"
 	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge."
```

Comment:
You should probably change the name and description of this to reflect the fact it's empty.